src/jit/gentree.h

```diff
@@ -2516,7 +2526,7 @@ struct GenTreeCall final : public GenTree
     // TODO-ARM: enable this for HFA returns on Arm64 and Arm32
     // TODO-X86: enable this for long returns on x86
     // TODO-AllArch: enable for all call nodes to unify single-reg and multi-reg returns.
-#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
+#ifdef FEATURE_MULTIREG_RET
     ReturnTypeDesc    gtReturnTypeDesc;
 
```

Comment:
Changing this to FEATURE_MULTIREG_RET will enable this for arm64 as well, right? Is that the right behavior at this point?